torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -1541,15 +1529,20 @@ PythonArgs PythonArgParser::raw_parse(
     PyObject* parsed_args[]) { // NOLINT
   if (signatures_.size() == 1) {
     auto& signature = signatures_[0];
-    signature.parse(self, args, kwargs, parsed_args, true);
+    std::vector<py::handle> overloaded_args;
```

Comment:
py::handle? We don't own these objects here? Who keeps them alive?

Reply:
Two explanations:

1) The args tuple and kwargs dict own references to the arguments. They are valid for at least as long as the function call.
2) This is how it worked before, so if it worked then it will keep working now!